CHANGELOG.md

```diff
@@ -102,10 +103,18 @@ The entries below are relative to the `zcash_primitives::sapling` module as of
     generic parameters and returns `(UnauthorizedBundle, SaplingMetadata)`. The
     caller can then use `Bundle::<InProgress<Unproven, _>>::create_proofs` to
     create spend and output proofs for the bundle.
+  - `SaplingBuilder::build` now takes a `BundleType` argument that instructs
+    it how to pad the bundle with dummy outputs.
+  - `SaplingBuilder::bundle_output_count` has been changed to use a padding
+    rule to compute its result. It also now returns a `Result<usize, &'static str>`
```

Comment:
```suggestion
  - `SaplingBuilder::bundle_output_count` has been changed to use a bundle
    type to compute its result. It also now returns a `Result<usize, Error>`
```
This first sentence would be unnecessary if `SaplingBuilder` were constructed with the bundle type.